cairo/tests/utils/strategies.py

```diff
@@ -226,6 +234,57 @@ def tuple_strategy(thing):
     }
 ).map(lambda x: Evm(**x))
 
+## Strategies for block.py
+BLOCK_HASHES_LIST = [Hash32(Bytes32(bytes([i] * 32))) for i in range(256)]
+
+environment_lite_block = st.integers(
```

Comment:
why not put this in evm_lite directly?

Reply:
>  strictly minimal for the block.py test

this might become a problem down the line if we test the entire flow and we have an inconsistency betweem the block_hash_list and the current block number. the default strategy should ensure this consistency is respected

Reply:
will try to put the strategies again to see if we still get data_too_large

Reply:
done 
The idea of generating a int that would be tranformed into a hash32 is raising data_to_large though.